YUViewLib/src/statistics/StatisticsData.cpp

```diff
@@ -35,6 +35,7 @@
 #include "common/functions.h"
 
 #include <QPainterPath>
+#include <QtGui/QPolygon>
```

Comment:
I removed them. Basically took the corresponding code from qt and removed/change the types to std types. Is this a problem with licensing? Think not since its open source and we already mention that we use it.

Reply:
There is still some QPoint, QStringList, QRect. I can remove those as well.

Reply:
Hmm you are raising a good point actually. Qt is LGPL I think and our license is GPL. Not sure if this is compatible/lega.

Reply:
I found a source on stackoverflow (see comment in code) which I used instead. I also had to optmized that quite a bit, so now it should be different enough